proto/store.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+option go_package = "github.com/momentohq/client-sdk-go;client_sdk_go";
+option java_multiple_files = true;
+option java_package = "grpc.store";
+option csharp_namespace = "Momento.Protos.Store";
+
+package store;
+
+service Store {
+  rpc Get(_GetRequest) returns (_GetResponse) {}
+  rpc Set(_SetRequest) returns (_SetResponse) {}
+  rpc Delete(_DeleteRequest) returns (_DeleteResponse) {}
+}
+
+message _GetRequest {
+  bytes key = 1;
+  string store_id = 2;
+}
+
+message _GetResponse {
+  oneof result {
+    Miss miss = 1;
+    Hit hit = 2;
+  }
+  message Miss { }
+  message Hit {
+    bytes value = 1;
```

Comment:
do we want to explicitly model other data types here while we have the chance? There has been a lot of consternation about how in the cache proto we didn't model numeric values in the proto and that forced us to do weird things with them, like requiring them to be strings and then calling increment on strings. I think that if we had to do over with again we probably would have done a `oneof` for the values, and supported string/bytes/int/float as discrete types. cc: @kvcache

Reply:
like this? https://github.com/momentohq/client-protos/pull/277/commits/fff5d408ea9d3d7d16bc46e3359ec5087051d19a